modules/core/src/controllers/controller.js

```diff
@@ -137,12 +142,7 @@ export default class Controller {
     if ('onStateChange' in props) {
       this.onStateChange = props.onStateChange;
     }
-    if ('viewState' in props) {
```

Comment:
No longer possible to override? If I set a new `viewState` shouldn't controllers adapt? Or handled on ViewManager level?

Reply:
Yes, `viewState` is flattened by the viewManager.